test/service/src/lib.rs

```diff
@@ -157,7 +161,7 @@ pub fn new_partial(
 
 	let import_queue = cumulus_client_consensus_relay_chain::import_queue(
 		client.clone(),
-		client.clone(),
+		block_import,
```

Comment:
You probably also want to return this from `new_partial`

Reply:
Done in https://github.com/paritytech/cumulus/pull/1782/commits/90329ffd657df12ce560881e0fa719f13985927b